cfg/config.go

```diff
@@ -85,6 +85,8 @@ type FileCacheConfig struct {
 	MaxSizeMb int64 `yaml:"max-size-mb,omitempty"`
 
 	ParallelDownloadsPerFile int64 `yaml:"parallel-downloads-per-file,omitempty"`
+
+	WriteBufferSize int64 `yaml:"write-buffer-size,omitempty"`
```

Comment:
this will not be documented anywhere and is just a safety net incase something goes wrong. Yes, this can be configured anything in multiple of 4096 (4KiB)